File: NMAP/read_nmap_ms17010.py
#!/usr/bin/python

import sys
import xml.etree.ElementTree as xml

fileName = sys.argv[1]

tree = xml.parse(fileName)
root = tree.getroot()

for hosts in root.findall('host'):
    if (hosts.find('hostscript')):
        ip = hosts.find('address').get('addr')
        print ip, ", this host is Vulnerable to smb-vuln-ms17-010."

File: NMAP/read_nmap_ms17010.php
<?php

$fileName = $argv[1];

$xml = simplexml_load_file($fileName);

foreach ($xml->host as $hosts) {
  if(!empty($hosts->hostscript->script["id"])){
    echo $hosts->address["addr"] . ", this host is Vulnerable to ". $hosts->hostscript->script["id"] . ".";
    echo "\n";
  }
}
?>
